go.mod

```diff
@@ -12,7 +12,6 @@ require (
 	github.com/jimstudt/http-authentication v0.0.0-20140401203705-3eca13d6893a
 	github.com/klauspost/cpuid v1.2.0
 	github.com/kylelemons/godebug v0.0.0-20170820004349-d65d576e9348 // indirect
-	github.com/lucas-clemente/quic-go v0.13.1
```

Comment:
they are using 0.29.2 in their repo...Is it possible we only have to update the version and we don't have to remove this support? 

They say they support go 1.19 on their github page:
https://github.com/lucas-clemente/quic-go

Reply:
same goes for here.